nvflare/private/fed/server/job_runner.py

```diff
@@ -385,8 +385,8 @@ def run(self, fl_ctx: FLContext):
                     )
 
                     if ready_job:
-                        reload_job = job_manager.get_job(ready_job.job_id, fl_ctx)
-                        if reload_job.meta.get(JobMetaKey.STATUS) != RunStatus.SUBMITTED:
+                        if self._check_job_status(job_manager, ready_job.job_id, RunStatus.SUBMITTED, fl_ctx):
```

Comment:
It took me a moment to understand that "_check_job_status" actually checks whether the job is NOT in the specified status. We should rename this function to something like _job_not_in_status(), so that the code can be more readable.

Reply:
Let's fix it later so we can build a RC tonight.